### ISSUE TITLE
Create preliminary roadmap; finalize Q4 perf OKRs

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/perf-frontend-issues#8]
### Why/User Benefit/User Problem
Eric and Vicky raised concerns about the OKRs we selected for Q4 in that they may not align with our release criteria: we should finalize them by the end of next week (10/18).

### Acceptance Criteria (how do I know when I’m done?)
- Q4 front-end perf OKRs are finalized
- Docs are updated in the necessary places

**Comment by mcomella** [comment]
After discussing with the group, the concerns are:
- wrt "warm start" objective
  - we have no definition of "warm start" on Android or how we can replicate it in automation
  - we have no way to measure whatever we define "warm start" to be ^
  - warm start is not part of the release criteria (but maybe it should be: we haven't measured how often it happens nor do we have a way to do so)
- wrt infrastructure objectives
  - Do we have enough capacity to stand up new perf tests and make perf improvements?
  - Do we have all the tests we need [for monitoring release criteria]? Is that higher priority than not regressing? (there may be some updates for this by the end of next week)

There were also a few other interesting thoughts:
- "Are there more pressing performance issues to solve [than warm start], e.g. via user feedback?"
- Maybe we should accept regressions for now, building out our telemtery and infra, and do perf improvements in H1 when things are more stable

We decided **the action item would be to set OKRs that:**
- Monitor the health of things we think are important (via e.g. telemetry, tests) like what's in the release criteria
- Ensure the release criteria has everything it needs and doesn't have things it doesn't need (via e.g. adding new telemetry, analyzing existing telemetry)

We may also wish to have an OKR for our test framework follow-up: #3.

**Comment by mcomella** [comment]
> * Monitor the health of things we think are important (via e.g. telemetry, tests) like what's in the release criteria
> * Ensure the release criteria has everything it needs and doesn't have things it doesn't need (via e.g. adding new telemetry, analyzing existing telemetry)

To make this more concise, we want OKRs that:
- Monitor the health of important things, i.e. the release criteria
- Validate that we have the correct release criteria (has everything it needs and nothing it doesn't)

**Comment by mcomella** [comment]
We met again to discuss OKRs. Notes and my interpretations:
- we'll need to finalize our release criteria #9 before we can set final OKRs that address it: we may need to keep updating these beyond the next week
- we could consider having a generic OKR (e.g. institute 2 tests based on release criteria) to compensate for incompleteness
- we may wish to have an OKR for finishing the release criteria
- We may not want to set OKRs that have an expectation of not regressing (i.e. the current OKRs). However, we can have an OKR to start instituting a process where we check the regressions created by new features and decide if we want to accept the regression or not

**Comment by mcomella** [comment]
Waiting on more finalized release criteria before continuing.

**Comment by mcomella** [comment]
Requirements have been changing so I don't think this bug is totally actionable or helpful at the moment.
